server/storage/rethinkdb.go

```diff
@@ -0,0 +1,205 @@
+package storage
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/dancannon/gorethink"
+	"github.com/docker/notary/storage/rethinkdb"
+)
+
+// RDBTUFFile is a tuf file record
+type RDBTUFFile struct {
```

Comment:
I want to break up the storage package into sub packages that define specific backends. Then we'd have a `storage/rethinkdb/models.go` but for now, the current models file is gorm models and I don't want to mix them up.